chapters/annotations.tex

```diff
@@ -109,10 +109,11 @@ \subsection{Annotations for Figures}\label{annotations-for-figures}
 The \lstinline!identifier! in \lstinline!Figure! and \lstinline!Plot! is an optional \lstinline!String! identifier, and is intended to identify the
```

Comment:
I actually think that the proposed change works as a replacement for the _optional_ here.  What about dropping the word _optional_, and instead put the added sentence below as the second one of the paragraph (also changing _the uniqueness requirement_ to just _uniqueness requirements_)?

Reply:
ok for dropping optional.